target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -72,6 +72,11 @@ case "$board" in
 			macaddr_setbit_la "$base_mac" > /sys${DEVPATH}/macaddress
 		fi
 		;;
+	dlink,dir-3040-a1)
```

Comment:
Please move it up and place it after case `dlink,dap-x1860-a1`.

Reply:
Handled with new amendment